### PR TITLE
[Notifier] Telnyx

### PR DESCRIPTION
**Pull request by StaffNowa**

### CODE REVIEW

File: notifier.rst [hunk 1]
```diff
@@ -75,6 +75,7 @@ Sinch           ``symfony/sinch-notifier``            ``sinch://ACCOUNT_ID:AUTH_
 Smsapi          ``symfony/smsapi-notifier``           ``smsapi://TOKEN@default?from=FROM``
 SmsBiuras       ``symfony/sms-biuras-notifier``       ``smsbiuras://UID:API_KEY@default?from=FROM&test_mode=0``
 SpotHit         ``symfony/spothit-notifier``          ``spothit://TOKEN@default?from=FROM``
+Telnyx          ``symfony/telnyx-notifier``           ``telnyx://API_KEY@default?from=FROM&messaging_profile_id=MESSAGING_PROFILE_ID``
 Twilio          ``symfony/twilio-notifier``           ``twilio://SID:TOKEN@default?from=FROM``
 ==============  ====================================  ===========================================================================
```
Comment (OskarStark) [inline notifier.rst]: Please add a versionadded directive for `5.4` below

**Review by OskarStark — CHANGES_REQUESTED** [review]
verisonadded missing

**Comment by StaffNowa** [comment]
@OskarStark done )

File: notifier.rst [hunk 2]
```diff
@@ -92,6 +92,10 @@ Twilio          ``symfony/twilio-notifier``           ``twilio://SID:TOKEN@defau
     The Iqsms, GatewayApi, Octopush, AllMySms, Clickatell, SpotHit, FakeSms, LightSms, SmsBiuras
     and MessageBird integrations were introduced in Symfony 5.3.
 
+.. versionadded:: 5.4
+
+    The Telnyx integrations were introduced in Symfony 5.4.
```
Comment (OskarStark) [inline notifier.rst]:
```suggestion
    The Telnyx integration was introduced in Symfony 5.4.
```

Comment (StaffNowa) [reply notifier.rst]: Fixed. Updated

**Review by OskarStark — APPROVED** [review]

**Comment by StaffNowa** [comment]
Merged https://github.com/symfony/symfony/commit/2fb272ef85dedbe1d55c33cc3253e7b15e454f4a :)))

**Review by OskarStark — APPROVED** [review]

**Comment by OskarStark** [comment]
Can you please rebase? Thanks

**Comment by StaffNowa** [comment]
@OskarStark solved ;)

**Review by OskarStark — APPROVED** [review]

**Comment by javiereguiluz** [comment]
Thank you Vasilij.

**Pull request merged**